### ISSUE TITLE
issue with aligned_alloc on macOS 10.12 with clang 802.0.42 cannot find <malloc.h>

### ISSUE THREAD

**Issue by samhodge** [google/gemmlowp#134]
It seems that under /usr/include malloc.h is at /usr/include/malloc/malloc.h

I thought we could use stdlib.h instead, but after some reading I found that this has other issues.

Anyway the use case was to compile tensorflow r1.6 from tensorflow/contrib/cmake on macOS 10.12 with cland 802.0.42 and everything worked apart from gemmlowp so I thought I ought to let you know.

**Comment by bjacob** [comment]
I think that's part of what this fixed:
https://github.com/google/gemmlowp/commit/f4763b640a2f20dccfdaa6f545f59c2c7f34c502#diff-1d9ff34501cff9c205f20799e7a497db

Is TensorFlow still referencing an earlier gemmlowp commit? That would be a TF issue.

**Comment by samhodge** [comment]
Yeah for sure I hacked most of those changes in myself. I am on tensorflow 1.6 contrib cmake build so I guess I can put PR for that to be updated, just wanted to make sure that it was reported.
